ts/output/common/Wrappers/XmlNode.ts

```diff
@@ -150,74 +155,142 @@ export function CommonHtmlNodeMixin<
-  implements CommonHtmlNode<N, T, D, JX, WW, WF, WC, CC, VV, DD, FD, FC> {
+  return class CommonXmlNodeMixin extends Base
+  implements CommonXmlNode<N, T, D, JX, WW, WF, WC, CC, VV, DD, FD, FC> {
+
+    /**
+     * Don't set up inline-block styles for this
+     */
+    public static autoStyle = false;
+
+    /**
+     * @override
+     */
+    public static styles: StyleList = {
+      'mjx-measure-xml': {
+        position: 'absolute',
+        left: 0, top: 0,
+        display: 'inline-block',
+        'line-height': 'normal',
+        'white-space': 'normal'
+      },
+      'mjx-html': {
+        display: 'inline-block',
+        'line-height': 'normal',
+        'text-align': 'initial',
+        'white-space': 'initial'
+      },
+      'mjx-html-holder': {
+        display: 'block',
+        position: 'absolute',
+        top: 0, left: 0, bottom: 0, right: 0
+      }
+    };
+
+    /**
+     * @override
+     */
+    public rscale: number;
+
+    /**
+     * @override
+     */
+    public constructor(factory: WF, node: MmlNode, parent: WW = null) {
+      super(factory, node, parent);
+      this.rscale = this.getRScale();
+    }
 
     /**
      * @override
      */
     public computeBBox(bbox: BBox, _recompute: boolean = false) {
-      const hdw = this.getHDW((this.node as HtmlNode<N>).getHTML() as N, 'use', 'force');
-      const {h, d, w} = (hdw ? this.splitHDW(hdw) : this.jax.measureXMLnode(this.getHTML()));
+      const xml = (this.node as XMLNode).getXML() as N;
+      const hdw = this.getHDW(xml, 'use', 'force');
+      const {h, d, w} = (hdw ? this.splitHDW(hdw) : this.measureXmlNode(xml));
       bbox.w = w;
       bbox.h = h;
       bbox.d = d;
     }
 
     /**
-     * @return {N}   The HTML for the node
+     * @override
      */
     public getHTML(): N {
       const adaptor = this.adaptor;
-      const jax = this.jax;
-      const styles: StyleList = {};
-      const html = this.addHDW(adaptor.clone((this.node as HtmlNode<N>).getHTML() as N), styles);
-      const metrics = jax.math.metrics as ExtendedMetrics;
-      if (metrics.scale !== 1) {
-        styles['font-size'] = jax.fixed(100 / metrics.scale, 1) + '%';
+      let html = adaptor.clone((this.node as XMLNode).getXML() as N);
+      const styles = this.getFontStyles();
+      const hdw = this.getHDW(html, 'force');
+      if (hdw || this.jax.options.scale !== 1) {
+        html = this.addHDW(html, styles);
       }
-      const parent = adaptor.parent(jax.math.start.node);
-      styles['font-family'] = this.parent.styles?.get('font-family') ||
-        metrics.family || adaptor.fontFamily(parent) || 'initial';
       return this.html('mjx-html', {variant: this.parent.variant, style: styles}, [html]);
     }
 
     /**
-     * @param {N} html            The html to adjust if using or forcing HDW
-     * @param {StyleList} styles  The styles object to add to, as needed
+     * Implemented in subclasses
+     * @override
```

Comment:
Should that be an abstract method?

Reply:
I have made the `CommonXmlNodeMixin` be an abstract class and this an abstract method.  I wonder if **all** the common mixin classes should be abstract (as they will be subclasses to SVG and CHTML specific versions)?

The `XMLNode`  itself, however, can't be abstract, as it is instantiated in the MathML input jax as the contents (child) of an `annotation-xml` node.